docs/architecture/Overview.md

```diff
@@ -0,0 +1,164 @@
+
+An application that leverages one of the composites looks as follows:
+
+        ┌─────────────────────────────────────────────┐
+        │ Contoso Application                         │
+        │  * Create adapters                          │
+        │  * Create composite using the adapters      │
+        │   ┌─────────────────────────────────────┐   │
+        │   │ UI Composites                       │   │
+        │   │  * CallComposite                    │   │
+        │   │  * ChatComposite                    │   │
+        │   │  * CallWithChatComposite            │   │
+        │   │    ┌─────────────────────────────┐  │   │
+        │   │    │ UI Components               │  │   │
+        │   │    └─┬───────────────────────▲───┘  │   │
+        │   │      │                       │      │   │
+        │   │    ┌─┼───────────────────────┼───┐  │   │
+        │   │    │ │   Component Bindings  │   │  │   │
+        │   │    │ │                       │   │  │   │
+        │   │    │Handlers            Selectors│  │   │
+        │   │    └─┬───────────────────────┬───┘  │   │
+        │   │      │                       │      │   │
+        │   └──────┼───────────────────────┼──────┘   │
+        │          │                       │          │
+        │   ┌──────┴───────────────────────┼──────┐   │
+        │   │ Adapters                     │      │   │
+        │   │  * CallAdapter               │      │   │
+        │   │  * ChatAdapter               │      │   │
+        │   │  * CallWithChatAdapter       │      │   │
+        │   │      │                       │      │   │
+        │   │    ┌─▼───────────────────────┼───┐  │   │
+        │   │    │ StatefulClients         │   │  │   │
+        │   │    └─────────────────────────┴───┘  │   │
+        │   │                                     │   │
+        │   └─────────────────────────────────────┘   │
+        │                                             │
+        │                                             │
+        └─────────────────────────────────────────────┘
+
+_edit on [asciiflow](https://asciiflow.com/#/share/eJzNlcFOwzAMhl%2FFyhGtl4GE2G30ws4T4pJLaAONlCVV40qbpr0F2oNwRDwNT0ImttKVJe0KLY2syJXyOc7v1FkTxRacTEgupHgMWBYlQS6CSC9SbQRyQ0ZEshXP7JI1JUtKJjdXlyNKVtYbX4%2Bth3yJ9oOSj5f34Rilys4QaoXaaJimqRQRQ6EVuMY3BRcQZpwhBxazFHlmnNBJqpAPciPUM2BSjVSi4F%2F0%2BZkD3M%2BsXIe6u097gtydnElZ0OeRCcOWpN3zQWDijOAgO5bcv28hs%2BIKqzLX0n9p29eGWZccR3GGqvWXW8gNt0LF9oc0JaxRBO%2B19Ee4YyqWlQ4y55JHqDPTa70bK9e%2B3v1bJYdqlu78YYDZ7%2Bap78Xxq3%2FoifsQ57O2j7ZmS734OEYd27BeTnov5vatmwL59oU52uf%2BKZehFEfdvLSwLu%2BuLpZH7ZpxkuzfKjk0H7%2BkhmOUbMjmE1qM9VQ%3D)_
+
+The composite API is split into two parts. The composite proper corresponds to the UI components from the lower-level API.
+- The composite consists of a single React component that renders the entire composite throughout the experience.
```

Comment:
This implementation detail in unnecessary and can be skipped I believe. Since every react app ultimately is a single component...

Reply:
I wanted to contrast with the UI components where we exports lots of components. Tried to clarify by rewording a bit.